platform/iphone/os_iphone.cpp

```diff
@@ -107,13 +107,12 @@ Error OSIPhone::initialize(const VideoMode &p_desired, int p_video_driver, int p
 	RasterizerGLES3::register_config();
 	RasterizerGLES3::make_current();
 
-	visual_server = memnew(VisualServerRaster());
-	/*
-		FIXME: Reimplement threaded rendering? Or remove?
+	// FIXME: Reimplement threaded rendering? Or remove?
```

Comment:
That comment is no longer relevant since this commit does reimplement it. Same in Haiku and UWP code.